packages/mobile/src/firebase/firebase.ts

```diff
@@ -382,15 +367,20 @@ export async function readOnceFromFirebase(path: string) {
     .then((snapshot) => snapshot.val())
 }
 
-export async function setUserLanguage(address: string, language: string | null) {
+export async function setRegistrationProperties(address: string, properties: Record<string, any>) {
```

Comment:
I think we should be explicit about what props we allow here to prevent mistakes:
```suggestion
export async function setRegistrationProperties(address: string, properties: { language?: string | null, fcmToken?: string, appVersion?: string}) {
```

Reply:
yeah good idea :D applied